hatchery/ram.go

```diff
@@ -44,6 +44,10 @@ func (creds *CREDS) acceptTGWShare(ramArn *string) error {
 	if len(resourceShareInvitation.ResourceShareInvitations) == 0 {
 		// Log that there are no invitations
 		Config.Logger.Printf("No invitations found something fishy is going on")
+		err := creds.acceptTGWShare(ramArn)
+		if err != nil {
+			return err
+		}
```

Comment:
can you explain this change?

Reply:
I've refactored this bit, because it took me a bit to understand what was happening here as well. 

Hopefully the new update makes it easier to understand.